README.md

```diff
@@ -9,3 +9,15 @@ The default Bazel registry for the Bzlmod external dependency system of Bazel. I
 ## Contributing
 
 To contribute, check our [BCR  policies](docs/bcr-policies.md) and [contribution guidelines](docs/README.md).
+
+## Depending on the BCR infrastructure (and how not to)
+
+The core infrastructure of the Bazel Central Registry depends on Github and Google Cloud. Bazel users who enable Bzlmod by default depend on the BCR.
+While we try to keep the BCR infrastructure simple and reliable, we're not responsible for any damages caused by build failures due to potential BCR infrastructure failures.
+
+If you consider it's necessary, you can do the following to avoid depending on the BCR infrastructure while still making use of the information checked in the BCR.
+
+- Clone the BCR repository or mirror the content to your own infrastructure and use `--registry` option to change the default Bazel registry to your own.
```

Comment:
link `--registry` to: https://bazel.build/reference/command-line-reference#flag--registry

Reply:
Done.